.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   version_tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
This pins the ubuntu version as `ubuntu-latest` does not support certain OTP versions.

See [this issue](https://github.com/erlef/setup-beam/issues/161#issuecomment-1336129218)